src/viam/sdk/services/mlmodel/server.cpp

```diff
@@ -51,11 +51,7 @@ ::grpc::Status MLModelServiceServer::Infer(
 
     std::shared_ptr<MLModelService> mlms = std::dynamic_pointer_cast<MLModelService>(rb);
 
-    if (request->has_input_data() && request->has_input_tensors()) {
-        return {::grpc::StatusCode::INVALID_ARGUMENT, "Called [Infer] with both forms of input"};
-    }
-
-    if (!request->has_input_data() && !request->has_input_tensors()) {
+    if (!!request->has_input_tensors()) {
```

Comment:
I think there is an extra `!` here.

Reply:
...coding sans coffee. Thanks! This was responsible for the test failure.